File: Prod.RutaDigital.Entidades/Pregunta/Pregunta.cs
namespace Prod.RutaDigital.Entidades;

public class Pregunta
{
    public int id_pregunta { get; set; }
    public int id_modulo { get; set; }
    public int id_tipo_test { get; set; }
    public int id_recomendacion { get; set; }
    public int id_tipo_respuesta { get; set; }
    public int orden { get; set; }
    public string? codigo { get; set; }
    public string? descripcion { get; set; }
}

File: Prod.RutaDigital.Presentacion.Angular/ClientApp/src/app/autodiagnostico/interfaces/pregunta.ts
import { Respuesta } from './respuesta';
import { TipoRespuesta } from '../../shared/enums/tipo-respuesta.enum';
import { Alternativa } from './alternativa';

export interface Pregunta {
  id_pregunta?: number;
  id_modulo?: number;
  id_tipo_test?: number;
  id_recomendacion?: number;
  id_tipo_respuesta?: TipoRespuesta;
  orden?: number;
  codigo?: string;
  descripcion?: string;

  respuestas?: Respuesta[];

  alternativas?: Alternativa[];
}
